ADRs/RH/SIG-SRE/ADR-00005 SLO Lifecycle.md

```diff
@@ -0,0 +1,131 @@
+
+Part of healthy SLO adoption is the periodic review of the service's SLO needs. These review tasks may include:
+
+* Do the service's SLOs still measure the right thing(s) to measure customer happiness and service reliability?
+* Are the SLOs' SLIs still appropriate? Are there newer, better (ex. less computationally expensive) indicators that should be used?
+* Is a SLO still needed? (ex. Does the service landscape or user experience still demand it? Has another SLO superseded it?)
+
+These steps can be used to evaluate the SLOs themselves, but there's also review on the targets set by the service's SLO(s):
+
+* Are the service's SLOs being met?
+  * If they're not, what are common reasons why not?
+* What are the common reason(s) for risks and observed hits to the error budget?
+  * How can they be mitigated in the future?
+* What is the error budget burn rate? Will corrective action be needed at the current rate?
+
+Remember that some of these review steps can be outwardly facing to include external stakeholders such as service consumers within the organization.
+
+#### Sprint Planning Hooks
+
+A primary goal of Service Level Objectives is to inform the natural cycle of work for product feature teams by adding an additional signal for use in planning work. The new signal is the service's remaining Error Budget.
+
+The "elevator pitch" flow chart looks like this: If there's error budget remaining, it's a green light to focus on features, otherwise, focus on reliability.
+
+```mermaid
+graph TD
+  A(Sprint Planning) --> B{Error Budget<br />Available?}
+  B --> |Yes|C[Features]
+  B --> |No|D[Reliability]
+  C & D --> E(Prioritise the appropriate work)
+```
+
+In reality, however, the error budget is not meant to be the _only_ signal feeding sprint planning; it is merely _another_ signal that can be used to prioritise work. Planning is still up to the team and can focus on features when it makes sense. Sprint planning may look like this with the error budget informing decisions.
+
+```mermaid
+  graph TD
+  A(Sprint planning) ---> B[[Review product<br />roadmap needs]]
+  C[/Remaining error<br />budget/] -.->  |Informs|D[[Backlog grooming]] & B
+  B --> D
+  D --> E[[Pull cards<br />into sprint]]
+  E --> F(Start sprint)
+```
+
+A parting thought for this section is to remember the purpose of the Service Level Objective and associated error budget is to model customer satisfaction. Use caution when deprioritizing reliability for features when the error budget is depleted, or projected to become exhausted. Those who are closest to the service will have the best understanding of any impact for delaying reliability enhancements.
+
+## Challenges
+
+* Teams may desire to adopt the guidelines in this document but lack the data to do so because they don't have any SLO, or a way to measure the remaining error budget.
+* Teams may face encounter difficulties scheduling reliability work by those who deprioritize it in favour of "feature work."
```

Comment:
Here I think we should take some of what @mh21 mentioned his team does, where they have an "informing" phase for an SLO which allows them time to calibrate, before graduating it to "enforcing".  Have I got that right, @mh21 ?

Reply:
Err, sorry or was this @twaugh group?

Reply:
I can't recall who, but that's a great idea! I'll add a section about a "permissive phase."